lib/org/chromium/webidl/BaseParser.js

```diff
@@ -48,6 +50,10 @@ foam.CLASS({
         return this.StringPS.create();
       },
     },
+    {
```

Comment:
Are we deliberately not committing to this having a particular type because it can be one of many things?